README.md

```diff
@@ -17,19 +17,18 @@ A Clojure-inspired atom is placed above the cursor composition, and is the only
 ```js
 const data = Immutable.fromJS({a: 1, b: 2});
 
-const cursor = Cursor.from(data, (prevValue, nextValue) => {
-  console.log(newData)
-  // First call: $> Map { "a": 2, "b": 2 }
-  // Second call: $> Map { "a": 2, "b": 3 }
+const cursor = Cursor.from(data, (nextValue, prevValue, keyPath) => {
+  console.log('Value changed from', prevValue, 'to', nextValue, 'at', keyPath);
 });
 
 // Multiple update transactions are serialised.
 cursor.set('a', 2);
+// Value changed from Map { a: 1, b: 2 } to Map { a: 2, b: 2 } at []
 cursor.set('b', 3);
+// Value changed from Map { a: 2, b: 2 } to Map { a: 2, b: 3 } at []
```

Comment:
I think the former is preferable, since it's good to highlight the update serialisation of multiple transactions